vmdk_plugin/main.go

```diff
@@ -35,22 +33,18 @@ import (
 )
 
 const (
-	pluginSockDir = "/run/docker/plugins"
-	mountRoot     = "/mnt/vmdk" // VMDK and photon volumes are mounted here
 	photonDriver  = "photon"
 	vmdkDriver    = "vmdk"
 	vsphereDriver = "vsphere"
 	defaultPort   = 1019
 )
 
-// An equivalent function is not exported from the SDK.
-// API supports passing a full address instead of just name.
-// Using the full path during creation and deletion. The path
-// is the same as the one generated internally. Ideally SDK
-// should have ability to clean up sock file instead of replicating
-// it here.
-func fullSocketAddress(pluginName string) string {
-	return filepath.Join(pluginSockDir, pluginName+".sock")
+// Server responds to HTTP requests from Docker.
+type Server interface {
```

Comment:
nit - `Server` is way too generic.  It is better to name something like `PluginRequestsServer` (or whatever you prefer, as long as it is more specific than `Server` (the nit still applies if it's an old code :-) )